README.md

```diff
@@ -12,15 +12,15 @@ Fury is a blazing-fast multi-language serialization framework powered by **jit(j
 https://furyio.org
 
 ## Features
-- **Multiple languages**: Java/Python/C++/Golang/Javascript/Rust.
-- **Zero-copy**: cross-language out-of-band serialization inspired
+- **Multiple languages**: Java/Python/C++/Golang/Javascript/Rust/Cython/Typescript.
```

Comment:
```suggestion
- **Multiple languages**: Java/Python/C++/Golang/JavaScript/Rust/Scala/TypeScript.
```

Reply:
Cython is just a compiler for Python, how about removing it. And fury support scala, I think we can add it instead.